sdk/client.go

```diff
@@ -0,0 +1,186 @@
+package sdk
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"mime/multipart"
+	"net/http"
+	"net/url"
+	"sync"
+
+	"github.com/root-gg/plik/server/common"
+)
+
+var (
+	userAgent = "Plik Golang SDK v1.0.0"
```

Comment:
If I remember well, you already have a version for the server, why not reuse it ?

Reply:
You are right, it's not the best idea to put the version like this.

But, it's not linked with Plik server versionning